core/src/impl/Kokkos_ViewTracker.hpp

```diff
@@ -89,6 +91,17 @@ struct ViewTracker {
   KOKKOS_INLINE_FUNCTION
   explicit ViewTracker(const track_type& tt) noexcept
       : m_tracker(tt, view_traits::is_managed) {}
+
+  //! Move constructor.
+  KOKKOS_INLINE_FUNCTION
+  ViewTracker(ViewTracker&& other) : m_tracker(std::move(other.m_tracker)) {}
+
+  //! Move assignment.
+  KOKKOS_INLINE_FUNCTION
+  ViewTracker& operator=(ViewTracker&& other) {
+    m_tracker = std::move(other.m_tracker);
+    return *this;
+  }
```

Comment:
Same here. How does the behavior differ from the `default` implementation?

Reply:
I applied your suggestion (using the `default`). Thanks!